crates/gpui/src/platform/windows/window.rs

```diff
@@ -926,3 +1084,6 @@ unsafe fn set_window_long(hwnd: HWND, nindex: WINDOW_LONG_PTR_INDEX, dwnewlong:
         SetWindowLongW(hwnd, nindex, dwnewlong as i32) as isize
     }
 }
+
+const DRAGDROP_GET_COUNT: u32 = 0xFFFFFFFF;
+const FILENAME_MAXLENGTH: usize = 256;
```

Comment:
Shouldn't this be `windows::Win32::Foundation::MAX_PATH`?

Reply:
That's right.